pkg/helm/install_chart_test.go

```diff
@@ -0,0 +1,32 @@
+package helm
+
+import (
+	"github.com/stretchr/testify/assert"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+	"io/ioutil"
+	"testing"
+)
+
+func TestInstallChart(t *testing.T) {
+	chart := "https://srv-file5.gofile.io/download/dtv19k/influxdb-3.1.1.tgz"
+	store := storage.Init(driver.NewMemory())
+	actionConfig := &action.Configuration{
+		Releases:     store,
+		KubeClient:   &kubefake.PrintingKubeClient{Out: ioutil.Discard},
+		Capabilities: chartutil.DefaultCapabilities,
+		Log:          func(format string, v ...interface{}) {},
+	}
+	_, err := InstallChart("test", "test", chart, actionConfig)
+	if err != nil {
+		t.Error(err)
+	}
+	rels, err := ListReleases(actionConfig)
+	if err != nil {
+		t.Error(err)
+	}
+	assert.Len(t, rels, 1, "Release list should contain installed chart release")
```

Comment:
checking the response size is not enough - what if `ListRelease` has a bug and returns only one release, no matter how many are there? You should check if returned release is really the one we have tried to install above.

Reply:
added few more checks, please check